File: solucoes/2020/nivel 1/fase 1/py/3por2.py
qtd = int(input())
valores = []
soma = 0
i = 0

while(i < qtd):
  nums = int(input())
  valores.append(nums)
  i += 1

valores.sort(reverse=True)

j = 2
while(j < qtd):
  #valores.pop(j)
  valores[j] = 0
  j += 3

for index in range(len(valores)):
  soma += valores[index] 
  
print(soma)

File: solucoes/2020/nivel 1/fase 1/js/3por2.js
var N, P;
var nums = [];
var soma = 0;

scanf('%d', "N");

function ordena(a, b){
    return a - b;
}

for(var i = 0; i < N; i++){
   scanf("%d", "nums[i]");
}

nums.sort(ordena);
nums.reverse();

for(var i = 1; i <= nums.length; i++){
	if(i% 3 !== 0){
       soma += nums[i-1];
    }
}

printf("%d", soma);